### ISSUE TITLE
Add a simple GUI! Hacktoberfest2019

### ISSUE THREAD

**Issue by ionicc** [codezoned/PyFry#7]

**Comment by SparshKat** [comment]
could you elaborate on what type of GUI you are looking for? does a good Bootstrap,CSS,JS,HTML file do the work for you?

**Comment by ionicc** [comment]
@SparshKat Absolutely, The user should be able to select the picture and also view the picture on the gui once the conversion is done 😄

**Comment by AzimPathan** [comment]
@ionicc should I do this for you?

**Comment by SparshKat** [comment]
@ionicc alright then! I'm on it :)

**Comment by SparshKat** [comment]
@ionicc could you elaborate a bit on how you want the website/webapp to operate, the readme doesn't tell much! Thanks :)

**Comment by gnany07** [comment]
Hi, I saw that the issue is open for more than 2 days .
I started working on it, had a simple gui which allows user to select an image and then we will show both original image and PyFried image... 😃 
Will be submitting PR by tommorow.

**Comment by gnany07** [comment]
Raised a pull request which will fix some minor bugs in PyFry.py and a basic gui.
I am a new-bie.
Please review and feel free to give comments.
